lib/transactions/maltego/nettacker_transforms/src/nettacker_transforms/transforms/wp_xmlrpc_scan.py

```diff
@@ -0,0 +1,49 @@
+import random
+
+from canari.maltego.transform import Transform
+from canari.maltego.entities import URL
+from canari.framework import EnableDebugWindow
+from common.entities import NettackerScan
+
+from lib.scan.wp_xmlrpc.engine import start
```

Comment:
@aman566 please change should be lib.vuln.wp_xmlrpc_dos - also remember to rename the file in Maktego transforms and please test with Maltego